### PR TITLE
spatialOntologyImplementation

### PR DESCRIPTION
**Pull request by Rahgooy**

### CODE REVIEW

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/LanguageBaseTypeSensors.scala [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        case _ => ""
+      }
+  }
+
+  def getSemanticRole(e: NlpBaseElement): String = {
+    val ta = getTextAnnotation(e)
+    val view = if (ta.hasView(ViewNames.SRL_VERB)) {
+      ta.getView(ViewNames.SRL_VERB)
+    } else {
+      null
```
Comment (kordjamshidi) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/LanguageBaseTypeSensors.scala:97]: maybe some warning messages as well?

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/SpRLSensors.scala [hunk 2]
```diff
@@ -99,12 +99,12 @@ object SpRLSensors extends Logging {
...
     var i = 0
     while (i < up.size - 1) {
-      path.append(getRelationName(relations, up(i), up(i + 1), "â†‘"))
+      path.append(getRelationName(relations, up(i), up(i + 1), "↑"))
       i += 1
     }
     i = down.size - 1
     while (i > 0) {
-      path.append(getRelationName(relations, down(i), down(i - 1), "â†“"))
+      path.append(getRelationName(relations, down(i), down(i - 1), "↓"))
```
Comment (kordjamshidi) [inline saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/SpRLSensors.scala:107]: why we have this and also the SpRLNewSensors? Maybe, remove that file and have all in here? We need to remove this also eventually when we have everything based on the base-types.

Comment (Rahgooy) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/SpRLSensors.scala:107]: yes, that is a legacy from our previous attempts.
After we finish with this, we'll convert all of those to use our new base types, and then we remove all that files

File: saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/LanguageBaseTypeSensors.scala (hunk 1 above)
Comment (Rahgooy) [reply saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/LanguageBaseTypeSensors.scala:97]: sure

**Comment by kordjamshidi** [comment]
This all looks good to me. If we have a couple of features for relations we can start doing the classifiers. We can merge this now if you want, and make the necessary organizations with the location of the files and apps with regard msprl. After that organization we can continue on completing the features and classifiers.

**Pull request merged**